src/components/UsernameEntry/UsernameEntry.js

```diff
@@ -4,8 +4,24 @@ import './UsernameEntry.css';
 import ListSelection from '../listSelection/ListSelection';
 
 class UsernameEntry extends React.Component {
-  sendReminder = async user => {
-    const result = await this.props.sendSlackMessage(user.id);
+  state = {
+    selectedUser: null
+  };
+
+  promptToConfirm = user => {
+    this.setState({selectedUser: user});
```

Comment:
It's not big, but we could rename the parameter to match the state name to use shorthand:

```
promptToConfirm = selectedUser => {
  this.setState({ selectedUser })
}
```